src/olympia/discovery/data.py

```diff
@@ -1,13 +1,23 @@
 # -*- coding: utf-8 -*-
 from django.utils.translation import string_concat, ugettext_lazy as _
 
+from olympia import amo
+
 
 class DiscoItem(object):
     def __init__(self, *args, **kwargs):
         self.addon_id = kwargs.get('addon_id')
         self.addon_name = kwargs.get('addon_name')
         self.heading = kwargs.get('heading')
         self.description = kwargs.get('description')
+        self.type = kwargs.get('type', amo.ADDON_EXTENSION)
+
+    def __str__(self):
+        return 'DiscoItem(%s, %s, %s, %s, %s)' % (
+            self.addon_id, self.addon_name, self.heading, self.description,
+            self.type)
```

Comment:
This is actually more `__repr__` than a `__str__` implementation, a `__str__` might look like `"{name} - {heading}: {description}".format(self.addon_name, self.heading, self.description)"` imho.

Reply:
I only added it to debug the tests, but can do.

Reply:
https://github.com/mozilla/addons-server/pull/6324/commits/5d695e8ac12962936d1e3bfedbbe321ad5d2709f changes it to be `__repr__`